.github/testflinger/job-def.yaml

```diff
@@ -0,0 +1,116 @@
+job_queue: dell-precision-3470-c30322
+provision_data:
+  distro: jammy
+  user_data: |
+    #cloud-config
+    # Note, much of this is borrowed from the
+    # charm-dev multipass template:
+    # https://github.com/canonical/multipass-blueprints/blob/main/v1/charm-dev.yaml
+
+    runcmd:
+    - |
```

Comment:
by curiosity, what is the rationale behind the putting the runcmd in given sections ?

Reply:
Well, it helped organize the sections for me at some point, but I think since then they've morphed and now seem very arbitrary. Now that you've pointed it out they are bothering me and I'm going to remove them :sweat_smile: